apps/gql/app/graphql/route.ts

```diff
@@ -17,10 +16,10 @@ const { handleRequest } = createYoga<KampusGQLContext>({
   schema: createSchema<KampusGQLContext>({ typeDefs, resolvers }),
   logging: "debug",
   graphiql: true,
-  context: async () => {
+  context: async ({ request }) => {
     const loaders = createLoaders(clients);
     const actions = createActions(clients);
-    const session = await getServerSession();
+    const session = await getSession(request);
```

Comment:
did i tell you how much i love you?????

Reply:
![](https://media.giphy.com/media/oFeUVZfiuim9G/giphy.gif)